web/src/theme/bootstrap-replacement.scss

```diff
@@ -0,0 +1,43 @@
+/* * * * * * * * IMPORTANT * * * * * *
```

Comment:
Can you explain why we need this class? I don't understand :)

Reply:
I created this file to replace the defaults, which bootstrap had and our frontend relied on.
These include basic things, like the size and spacing of some headings, and some other low level styling.
I thought about styling elements individually or putting this css in one of the other global styling files, but to be honest, i do not quite understand our global-css file structure.